src/TypeSpec.Extension/Emitter.Csharp/src/constants.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export const AzureDefaultAdditionalDecorators = [
+    "Azure\\.ClientGenerator\\.Core\\.@clientName"
```

Comment:
This is for testing. We should add the real Decorators Azure emitter want to emit, e.g. "Azure\\.ResourceManager\\.$armProviderNamespace"